src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -9,6 +9,8 @@
 */
 public class TorpedoStore {
 
+  Random generator = new Random();
```

Comment:
random variable made attribute of the class, instead of being defined in a function